pkg/features/features.go

```diff
@@ -157,7 +158,13 @@ func Features() map[FeatureID]Feature {
 		HelmValues: {
 			Title:       `Helm Values`,
 			Name:        `lovely_helm_values`,
-			Description: "This is a space separated list values files you'd like to use when rendering the helm chart. Defaults to `values.yaml` if that exists, but its fine if it doesn't. If you override this the file *must* exist. MERGE and PATCH will be applied to the first file in this list.",
+			Description: "This is a space separated list values files you'd like to use when rendering the helm chart. Defaults to `values.yaml` if that exists, but its fine if it doesn't. If you override this the file *must* exist (unless `LOVELY_HELM_IGNORE_MISSING_VALUE_FILES` is set). MERGE and PATCH will be applied to the first file in this list.",
```

Comment:
I'd probably like to reword this - and mention that MERGE/PATCH apply to the first used file.